checkout-core/src/main/java/com/adyen/checkout/core/internal/data/api/HttpClientExt.kt

```diff
@@ -66,7 +66,7 @@ suspend fun <T : ModelObject, R : ModelObject> HttpClient.post(
     adyenLog(AdyenLogLevel.VERBOSE) { "request - ${requestJson.toStringPretty()}" }
 
     val result = runAndLogHttpException { post(path, requestJson.toString(), queryParameters) }
-    val resultJson = JSONObject(String(result, Charsets.UTF_8))
+    val resultJson = JSONObject(String(result, Charsets.UTF_8).ifBlank { "{}" })
```

Comment:
Nit: `.ifBlank { "{}" }` seems to me a bit hacky (not sure this is the right word).
What if we create an empty `JSONObject()` when the string is blank?